helpers/pack_source_droidmedia-localbuild.sh

```diff
@@ -3,7 +3,7 @@ if [ ! -f ./out/target/product/${DEVICE}/system/lib/libdroidmedia.so ]; then
     exit 1
 fi
 
-pkg=droidmedia-0.0.0
+pkg=droidmedia-"${1:-0.0.0}"
```

Comment:
The actual changes in the script are NO-OP, so can be merged now and alternative instructions provided in the HADK FAQ for the time being